### PR TITLE
Add the manifest to build `dev-libs/libinput-*`.

### PR DESCRIPTION
**Pull request by Arignir**

### CODE REVIEW

File: dev-libs/libinput.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+@manifest(
+    name='libinput',
+    category='dev-libs',
+    description='''
+    A library that handles input devices for display servers and other applications that need to directly deal with input devices.
+    ''',
+    tags=['input', 'mouse', 'keyboard'],
+    maintainer='grange_c@raven-os.org',
+    licenses=[stdlib.license.License.MIT, stdlib.license.License.CUSTOM],
+    upstream_url='https://bitmath.org/code/mtdev/',
```
Comment (doom) [inline dev-libs/libinput.py:18]: Not sure that's quite the appropriate URL :p

File: dev-libs/libinput.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    kind=stdlib.kind.Kind.EFFECTIVE,
+    versions_data=[
+        {
+            'semver': '1.1.5',
```
Comment (doom) [inline dev-libs/libinput.py:22]: This version does not seem to match the one from the archive URL below, is it normal ?

Comment (Arignir) [reply dev-libs/libinput.py:22]: I may or may not have been half-asleep when writting this shit. Thanks.

**Comment by Arignir** [comment]
The version of libinput picked (0.5.0) i way too old. I'm gonna close this PR and open a new one with a more up-to-date version of libinput.

**Pull request closed without merging**